napari/_qt/qt_viewer.py

```diff
@@ -414,6 +419,29 @@ def on_key_release(self, event):
     def on_draw(self, event):
         """Called whenever drawn in canvas. Called for all layers, not just top
         """
+        if isinstance(self.view.camera, ArcballCamera):
+            self.viewer.camera._center = self.view.camera.center[::-1]
+            self.viewer.camera._scale = self.view.camera.scale_factor
+
+            # Do conversion from quaternion representation to euler angles
+            q = self.view.camera._quaternion
```

Comment:
This may be a nitpick, but I find it a bit confusing that we have both `self.viewer.camera` and `self.view.camera` in the same function.

Reply:
hmm, its not ideal, but not sure how to improve - maybe rename the `view` - this could come with the refactoring described in #855